packages/build/src/core/main.js

```diff
@@ -233,9 +234,20 @@ const initAndRunBuild = async function({
   logs,
   timers,
   testOpts,
+  buildbotServerSocket,
 }) {
   const constants = await getConstants({ configPath, buildDir, functionsDistDir, netlifyConfig, siteInfo, mode })
 
+  var buildbotClient
+  if (buildbotServerSocket) {
+    try {
+      buildbotClient = await startBuildbotClient(buildbotServerSocket)
```

Comment:
`closeBuildbotClient(buildbotClient)` should be called later on, for cleanup.
Maybe, at the same time as `stopPlugins()`?

```js
try {
  ...
} finally {
  await Promise.all([stopPlugins(childProcesses), closeBuildbotClient(buildbotClient)])
}
```